guides/v2.2/cloud/trouble/robots-sitemap.md

```diff
@@ -7,43 +7,47 @@ functional_areas:
 ---
 An attempt to generate and write the `sitemap.xml` file to the root directory results in the following error:
 
-```
+```terminal
 Please make sure that "/" is writable by the web-server.
 ```
 
 With {{site.data.var.ece}}, you can only write to specific directories, such as `var`, `pub/media`, `pub/static`, or `app/etc`. When you generate the `sitemap.xml` file using the Admin panel, you must specify the `/media/` path.
 
-You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the url: `<domain.your.project>/robots.txt`
+You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
 
 This requires ECE-Tools version 2002.0.12 and later with an updated `.magento.app.yaml` file. See an example of these rules in the [magento-cloud repository](https://github.com/magento/magento-cloud/blob/master/.magento.app.yaml#L43-L49).
 
-#### To generate a `sitemap.xml` file in version 2.2 and later:
+#### To generate a `sitemap.xml` file in version 2.2 and later
+
+1. Access the Magento Admin panel.
+1. On the _Marketing_ menu, click **Site Map** in the _SEO & Search_ section.
```

Comment:
I assume italicizing UI areas you don't click on is a cloud style thing?

Reply:
when doing a search for "menu,", i found other references to menus (Marketing x3, Extension x2, Content x2, Account x2). Marketing and Content have the ital formatting, Account and Extension do not.

Reply:
That's the style in Cloud, but I know that it is not consistently applied.

Reply:
There has been much discussion about what to do here. It's ongoing... I just wish we were consisten across repos/doc sets!

Reply:
leaving menu titles in ital for this topic